### PR TITLE
Community content

### PR DESCRIPTION
**Pull request by skim1420**

### CODE REVIEW

**Comment by skim1420** [comment]
@spinnaker/google

File: community/articles.md [hunk 1]
```diff
@@ -6,3 +6,34 @@ sidebar:
...
+* [How We Build Code at Netflix](http://techblog.netflix.com/2016/03/how-we-build-code-at-netflix.html), *Netflix Tech Blog*
+* [Deploying Node.js with Spinnaker](http://blog.greta.io/deploying-nodejs-with-spinnaker/), *Greta*
+* [Spinnaker Deployment Pipelines](http://riltsken.github.io/devops/infrastructure/deploymenttools/2016/02/08/spinnaker-deployment-pipelines.html), *Samuel Toriel*
+* [Setting up Spinnaker authentication with Okta and SAML](http://riltsken.github.io/devops/infrastructure/deploymenttools/2015/12/08/setup-okta-saml-with-spinnaker.html), *Samuel Toriel*
+* [Global Continuous Delivery with Spinnaker](http://techblog.netflix.com/2015/11/global-continuous-delivery-with.html), *Netflix Tech Blog*
+* [Moving from Asgard to Spinnaker](http://techblog.netflix.com/2015/09/moving-from-asgard-to-spinnaker.html), *Netflix Tech Blog*
+* [Netflix’s Spinnaker available now on Google Cloud Platform](https://cloudplatform.googleblog.com/2015/11/Netflixs-Spinnaker-available-now-on-Google-Cloud-Platform.html), *Gogole Cloud Platform Blog*
+
+
```
Comment (duftler) [inline community/articles.md]: Is the extra blank line here intentional?

**Review by duftler — APPROVED** [review]
LGTM.

File: community/contributing/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Filing issues
+
+Please file issues through the [Spinnaker GitHub issue tracker](https://github.com/spinnaker/spinnaker/issues).
+
+## How to Become a Contributor
+
+This is normally the place where we'd tell you about Contributor License Agreements and how you need to identify the right one and sign it before you are able to contribute code to Spinnaker.
+
+We'll probably end up doing something like this, but at the moment we're exploring options that might make it easier for most of you to contribute. Please stay tuned..
```
Comment (ttomsu) [inline community/contributing/index.md]: One or three periods, not two.

File: community/articles.md (hunk 1 above)
Comment (danielpeach) [reply community/articles.md]: s/Gogole/Google

File: community/contributing/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+In the meantime, if you're really excited about contributing and don't want to wait until we have all of this worked out, please reach out to us on [Slack](http://join.spinnaker.io). We really would love to accept your patches!
```
Comment (ttomsu) [inline community/contributing/index.md]: "We love accepting community patches!"

**Pull request merged**